libpod/networking_common.go

```diff
@@ -262,6 +262,12 @@ func (c *Container) getContainerNetworkInfo() (*define.InspectNetworkSettings, e
 				cniNet.Aliases = opts.Aliases
 				settings.Networks[net] = cniNet
 			}
+		} else {
+			settings.Networks = make(map[string]*define.InspectAdditionalNetwork, 1)
+			name := c.NetworkMode()
```

Comment:
Problem is that will not only return host and none. This can also result in `slirp4netns` which should be fine but `ns:/some/path` and `container:ID` might not be right. I don't know what docker does in this case, we should check this first.

Reply:
```
# cid=$(./bin/podman create alpine echo hi)
# ./bin/podman create --network=container:$cid alpine echo hi1
1a66aeb6e0bd6814c7a7dff9ef90847de88d85ccd4109d3e5ef2795eaffde82d
# ./bin/podman inspect --format '{{ .NetworkSettings.Networks }}' -l
map[podman:0xc0007059e0]
```
Versus
```
# docker create alpine echo hi
45410ef61a0d02a2433eab8f48beb2638be457fd92d3145c86829194471542f5
# docker create --network container:45410e alpine echo hi1
81820ea9f50c0894d39e48d9ca7d538cd584491ebcde8cd7af21b21ca5b751c8
# docker inspect --format '{{ .NetworkSettings.Networks }}' 81820ea9f
map[]
```

Reply:
But this is not changed by my code.

Reply:
yes it is, this is not what we showed before and not what docker does.
container: and ns: should not populate the map!

Reply:
Docker populates NS:
If I join a container to another containers network, why shouldn't the second container show the network of the first container?